react/components/useRuntime.ts

```diff
@@ -1,3 +1,28 @@
+import { RenderRuntime } from '../typings/runtime'
 import { useRuntime } from './RenderContext'
 
-export default useRuntime
+type PublicRuntime = Pick<
+  ReturnType<typeof useRuntime>,
```

Comment:
Why use `ReturnType<>` when we have the type available?

```suggestion
  RenderContextType,
```